coriolis/osmorphing/osmount/base.py

```diff
@@ -389,17 +389,22 @@ def _find_and_mount_root(self, devices):
         os_root_device = self._find_dev_with_contents(
             devices, all_files=files)
 
+        user_tips = (
+            "Please check the source VM's contents of '/etc/fstab' or try "
+            "using a more recent version of the worker's OS")
         if os_root_device is None:
             raise exception.OperatingSystemNotFound(
-                "root partition not found")
+                "Worker could not identify the migrated VM's root partition. "
```

Comment:
Coriolis was unable to identify the root partition of the OS being migrated for mounting during OSMorphing. Please ensure that the source VM's root partition(s) are *not* encrypted, and that they are using a filesystem type and version which is supported by the OS images used for the OSMorphing minion machine. Also ensure that the source VM's mountpoint declarations in '/etc/fstab' are all correct, and are declared using '/dev/disk/by-uuid/*' or 'UUID=*' notation. If all else fails, please retry while using an OSMorphing minion machine image which is the same OS release as the VM being migrated.